### ISSUE TITLE
Question regarding the training settings

### ISSUE THREAD

**Issue by Jianyi2004** [moojink/openvla-oft#32]
Hi, I have a question regarding the training settings.

I tried reducing the batch_size to 4 and found that on a single A100 40G GPU, the estimated training time is only 30 hours — which is twice as fast as using an A100 80G GPU (which takes around 60 hours).

I’m wondering:
Is it possible to reproduce the results reported in the paper using batch_size=4?
Or would a small batch size like this significantly affect performance?

Thanks in advance for your help!

**Comment by LukeLIN-web** [comment]
Why single A100 40G GPU is faster than single A100 80G GPU?

**Comment by moojink** [comment]
Hi @Jianyi2004, in general, a smaller batch size still works but will require more gradient steps to reach the same training loss. In most cases, we recommend training until the training L1 loss reaches ~0.01. If you can get there with a smaller batch size (and the loss is stable), it shouldn't really affect the policy's performance.
